From 4c3a31097970e551348102990b7c5ba684c68bbc Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Tue, 17 Mar 2015 15:42:29 +0100
Subject: [PATCH] virt-viewer: Clean up if no vm was chosen

It is safe to clean up when running virt-viewer without specifying
vm name if no vm was chosen. It brings back behavior before 88f6341.

The 'if (dom == NULL && err != NULL)' part was affected by commits
824c4b9, 1eaaf8c, 15c7d17 so the check for 'err' is not needed anymore.

(cherry picked from commit 66afac53b6769f079d537cfa130f212286981c93)
---
 src/virt-viewer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virt-viewer.c b/src/virt-viewer.c
index f077ab6..bcac2ca 100644
--- a/src/virt-viewer.c
+++ b/src/virt-viewer.c
@@ -745,7 +745,7 @@ virt_viewer_initial_connect(VirtViewerApp *app, GError **error)
                             &priv->domkey,
                             priv->conn,
                             &err);
-            if (dom == NULL && err != NULL) {
+            if (dom == NULL) {
                 goto cleanup;
             }
         }
